packages/core/src/utils.ts

```diff
@@ -27,3 +27,31 @@ export const rewriteRedirectTo = (
           : options?.redirectTo
       }
     : options
+
+export function getParameterByName(name: string, url?: string) {
+  if (!url) url = window?.location?.href || ''
```

Comment:
This will break in a Node environment. We should return if `window` is of the type `undefined`.
```ts
if (typeof window === 'undefined') {
  return
}

// ...
const { location } = window
```